apps/palette-tauri/src/styles.css

```
@@ -2427,8 +2670,9 @@ textarea {
 }
 
 .ask-compose .command-submit {
-  --ctl: 38px;
+  --ctl: 54px;
   --r-ctl: 10px;
+  flex: 0 0 var(--ctl);
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)